agent/logging.go

```diff
@@ -44,41 +43,33 @@ func getInc() int { return <-idSource }
 
 // GetSender configures the agent's local logging to a file.
 func (a *Agent) GetSender(ctx context.Context, prefix string) (send.Sender, error) {
-	var (
-		err     error
-		sender  send.Sender
-		senders []send.Sender
-	)
-
-	if os.Getenv(util.MarkerAgentPID) == "" { // this var is set if the agent is started via a command
-		if token := a.opts.SetupData.SplunkClientToken; token != "" {
-			info := send.SplunkConnectionInfo{
-				ServerURL: a.opts.SetupData.SplunkServerURL,
-				Token:     a.opts.SetupData.SplunkClientToken,
-				Channel:   a.opts.SetupData.SplunkChannel,
-			}
-			grip.Info("Configuring splunk sender.")
-			sender, err = send.NewSplunkLogger("evergreen.agent", info, send.LevelInfo{Default: level.Alert, Threshold: level.Alert})
-			if err != nil {
-				return nil, errors.Wrap(err, "creating Splunk logger")
-			}
-			senders = append(senders, sender)
+	var senders []send.Sender
+
+	if a.opts.SetupData.SplunkClientToken != "" && a.opts.SetupData.SplunkServerURL != "" && a.opts.SetupData.SplunkChannel != "" {
```

Comment:
Yes, it's as I described above - since the smoke test should never have a reason to be passed the Splunk credentials (and it currently doesn't), it would implicitly skip this block anyways.